### ISSUE TITLE
Several tricks to boost the performance of SCNN-Tensorflow in CULane

### ISSUE THREAD

**Issue by cardwing** [cardwing/Codes-for-Lane-Detection#5]
There are several tricks to boost the performance of SCNN-Tensorflow:

- Decrease the coefficient of the lane existence prediction loss
- Decrease the coefficient of the background pixels in the cross-entropy loss
- First pre-train the VGG-16 backbone on CULane. Then add the message passing module to VGG-16 and train the whole model jointly
- Introduce auxiliary tasks like drivable area detection and lane point regression
- Use a better optimizer, like SGD + Nesterov Momentum 
- Balance the proportion of different driving categories like normal, shadow and curve, in the training process
- Hard example mining
- Add weight decay
